CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### 1.0.20
+#### Additional Feature
+
+- Adding anchor id for all section types (sectionName-sectionId)
+- Showing anchor id under the title of each section in edit mode
```

Comment:
kindly update the readme to how it is now

Reply:
@jfayad I didn't get you on this one, your comment is on CHANGELOG file but it is about README file. Also I have pulled from master so the files has the last changes already

Reply:
typo, the changelog is outdated